### DISCUSSION TITLE
How can i download file in anonymous web

### DISCUSSION THREAD

**Discussion by Cricle** [minio/minio#15827] [Q&A] [answered]
I want to save in other server.
```html
<img src="http://xxx/bucket/fff.png"/>
```

But it alwaly return 401

**Comment by vadmeste** [comment, marked as answer]
@Cricle that's expected. By default, the access to buckets needs credentials. If you want to make your data in a given bucket public, take a look here 'mc anonymous -h'

**Reply by Cricle** [reply]
I alread set `public` like

![image](https://user-images.githubusercontent.com/26201412/194849479-47f26470-eb7a-44c6-b302-134925bfdbc2.png)

But when no any token access, it will be 401.

![image](https://user-images.githubusercontent.com/26201412/194849436-b56bb5d8-5670-4ab1-b880-a4fd00e5164b.png)


Have some method make access file without any token?

**Reply by Cricle** [reply]
Or should the server proxy file access?

**Reply by vadmeste** [reply]
@Cricle you need to use the S3 URL, not a Console URL.

The S3 url will be http://localhost:[your-s3-port]/ddd1/w.xlsx

**Reply by Cricle** [reply]
Thinks
